### PR TITLE
Update 02.introduction.md-the immune response

### PR DESCRIPTION
**Pull request by LSH2126**
### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/02.introduction.md [hunk 1]
```diff
@@ -121,6 +121,16 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
 #### Immunological Mechanisms of Coronavirus-driven Disease in Humans
 
 Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+One way that humans respond to viral threats is through the immune response.
+The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
+The innate immune system consists of barriers such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
+It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
+The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
+ These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@url:https://pubmed.ncbi.nlm.nih.gov/16455341 ].
```
Comment (agitter) [inline content/02.introduction.md:129]:
```suggestion
These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].
```

File: content/02.introduction.md [hunk 2]
```diff
@@ -121,6 +121,16 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
...
+
+After SARS-CoV-2 enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
+One of the main immune responses that contribute to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
+In addition to respiratory distress, this mechanism leads to organ failure and death in severe COVID-19 cases [@url:https://doi.org/10.1016/j.jpha.2020.03.001 ].
```
Comment (agitter) [inline content/02.introduction.md:133]:
```suggestion
In addition to respiratory distress, this mechanism leads to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].
```

File: content/02.introduction.md [hunk 3]
```diff
@@ -121,6 +121,16 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
 #### Immunological Mechanisms of Coronavirus-driven Disease in Humans
 
 Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
```
Comment (rando2) [inline content/02.introduction.md:123]:
```suggestion
```
We can get rid of this now that you've added real content!

File: content/02.introduction.md [hunk 4]
```diff
@@ -121,6 +121,16 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
...
+One way that humans respond to viral threats is through the immune response.
+The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
+The innate immune system consists of barriers such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
+It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
+The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
+ These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@url:https://pubmed.ncbi.nlm.nih.gov/16455341 ].
+
+After SARS-CoV-2 enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
```
Comment (rando2) [inline content/02.introduction.md:131]:
```suggestion
After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
```
I think this is just broadly the case, not SARS-CoV-2 specific, right?

File: content/02.introduction.md [hunk 5]
```diff
@@ -121,6 +121,16 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
 #### Immunological Mechanisms of Coronavirus-driven Disease in Humans
 
 Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+One way that humans respond to viral threats is through the immune response.
+The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
+The innate immune system consists of barriers such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
```
Comment (rando2) [inline content/02.introduction.md:126]:
```suggestion
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
```
I kept reading this as neutrophils and macrophages were also barriers -- I think the extra comma helps make it clear that the skin is the barrier you're talking about!

**Comment by rando2** [comment]
@nilswellhausen and @yemarshall, @LSH2126 is our new undergraduate researcher who has joined this project through the APSA summer program! Would you be able to provide some feedback for her on her summary? I think it's a lot better than #349 at least!

**Comment by yemarshall** [comment]
This all looks good & clear to me.

File: content/02.introduction.md (hunk 4 above)
Comment (yemarshall) [reply content/02.introduction.md]: yes, @rando2, this is broadly the case :+1:

**Pull request merged**